samples/core/Miscellaneous/Collations/Collations.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RootNamespace>EFCollations</RootNamespace>
+    <AssemblyName>EFCollations</AssemblyName>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="5.0.0-preview.3.20181.2" />
```

Comment:
You're too eager with your docs! ;-) These things aren't in preview 3...

Reply:
Yep, that's why this PR fails the build :) Just wanted to get this done with while everything is still fresh in mind.

We can hold off merging until preview4 is out...